packages/examples/src/app.css

```diff
@@ -21,6 +21,14 @@
     pre {
         white-space: pre-wrap;
     }
+    #incomingMessages { list-style-type: none; margin: 0; padding: 0; overflow: scroll; height: 20em;}
+    #incomingMessages li { padding: 0.1em 1%; }
+    #incomingMessages li:nth-child(odd) { background: #eee; }
+    #incomingMessagesContainer {
+        height: 20rem;
+        display: block;
+        border: 1px solid #ccc;
+    }
```

Comment:
Any reasons for not using Tailwind for these?

Reply:
I just copied some of this over from the old examples and then adjusted it to work, but good point - I will replace it with tailwind